examples/chart/teleport-cluster/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: teleport-cluster
 apiVersion: v2
-version: "10.0.0-dev"
+version: "10.0.1-dev"
```

Comment:
```suggestion
```

You can remove this edit, the version is automatically updated when we push releases.